src/uct/api/uct_def.h

```diff
@@ -299,6 +299,38 @@ typedef ucs_status_t (*uct_sockaddr_conn_request_callback_t)(void *arg,
                                                              size_t length);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Callback to fill the user's private data on the client side.
+ *
+ * This callback routine will be invoked on the client side before sending the
+ * transport's connection request to the server.
+ * It should be set by the client side when creating an endpoint.
+ * The user's private data should be placed inside the priv_data buffer and it
+ * will be sent to the server side.
+ * The maximal allowed length of the private data is indicated by the field
+ * max_conn_priv inside @ref uct_iface_attr.
+ * Other than communication progress routines, it is allowed to call other UCT
+ * communication routines from this callback.
```

Comment:
The sentence is tangled... change to "This callback may call communication progress routines as well as other UCT communication routines."